python/paddle/fluid/transpiler/inference_transpiler.py

```diff
@@ -387,6 +387,37 @@ def _fuse_batch_norm(self, program, place, scope):
         # And a better solution will be considered later.
         program = program.clone()
 
+    def _fuse_conv_sigmoid_mkldnn(self, program):
+        '''
+        Transpile the program by fused sigmoid activation into conv for MKLDNN program.
+        Sigmoid activation following convolution OP can be fused by adding
+        'fuse_sigmoid' attribute to convolution OP.
+        The result of fuse is:
+            - before:
+                - conv -> sigmoid ->any_other_op
+            - after:
+                - conv ->any_other_op
```

Comment:
Please remove the space between conv and arrow to keep it consistent with other comments.